cloud-controller-manager/cmd/digitalocean-webhook-server/Dockerfile

```diff
@@ -22,22 +22,23 @@ COPY go.mod go.mod
 COPY go.sum go.sum
 # cache deps before building and copying source so that we don't need to re-download as much
 # and so that source changes don't invalidate our downloaded layer
-COPY vendor/ vendor/
 
+COPY cloud-controller-manager/cmd/digitalocean-webhook-server/ cloud-controller-manager/cmd/digitalocean-webhook-server/
 
 # Copy the go source
 COPY  cloud-controller-manager/cmd/digitalocean-webhook-server/main.go main.go
 COPY  cloud-controller-manager/do cloud-controller-manager/do
 
-
 # Build
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -v -a -o doks_webhooks main.go
+RUN go mod vendor
```

Comment:
The container build failed referencing that running go mod vendor was required, I'll replicate again and grab the exact error.

Reply:
I still get the error after executing go mod tidy then vendor  prior to building the container. I think this is because I set -mod=vendor

```
#14 0.226       To ignore the vendor directory, use -mod=readonly or -mod=mod.
#14 0.226       To sync the vendor directory, run:
#14 0.226               go mod vendor
```